### ISSUE TITLE
Integration with Drupal

### ISSUE THREAD

**Issue by ajrobin** [janpaepke/ScrollMagic#205]
Having an issue with Scrollmagic in Drupal, I get this error triggered in the Jquery Once plugin when I call "new ScrollMagic();":

`Timestamp: 02/12/2014 09:25:38
Error: TypeError: Argument 1 of Node.contains does not implement interface Node.
Source File: https://dqqzjdqmiszdy.cloudfront.net/sites/default/files/js/js_0gj6QcpfRH2jzTbCQqf7kEkm4MXY0UA_sRhwPc8jC1o.js
Line: 3`

This is the code I'm using:

``` js
$(document).ready(function($) {
            function pathPrepare ($el) {
                  var lineLength = $el[0].getTotalLength();
                  $el.css("stroke-dasharray", lineLength);
                  $el.css("stroke-dashoffset", lineLength);
           }

var $stitch1top = $("path#stitch1top");
                var $stitch2top = $("path#stitch2top");
                var $stitch3top = $("path#stitch3top");
                var $stitch4top = $("path#stitch4top");
                var $stitch5top = $("path#stitch5top");

// prepare SVG
                pathPrepare($stitch1top);
                pathPrepare($stitch2top);
                pathPrepare($stitch3top);
                pathPrepare($stitch4top);
                pathPrepare($stitch5top);

                // init controller
                var controller = new ScrollMagic();

                // build tween

                var distancePerSecond = 500;

var tween = new TimelineMax();
                    var lineLength = $($stitch1top)[0].getTotalLength();
                    $($stitch1top).css("stroke-dasharray", lineLength);
                    $($stitch1top).css("stroke-dashoffset", lineLength);
                    tween.add(TweenMax.to($stitch1top, lineLength / distancePerSecond, {strokeDashoffset: 0, ease:Linear.easeNone}));
                var tween2 = new TimelineMax();
                    var lineLength2 = $($stitch2top)[0].getTotalLength();
                    $($stitch2top).css("stroke-dasharray", lineLength2);
                    $($stitch2top).css("stroke-dashoffset", lineLength2);
                    tween2.add(TweenMax.to($stitch2top, lineLength2 / distancePerSecond, {strokeDashoffset: 0, ease:Linear.easeNone}));
                    tween2.delay(1.9);
                var tween3 = new TimelineMax();
                    var lineLength3 = $($stitch3top)[0].getTotalLength();
                    $($stitch3top).css("stroke-dasharray", lineLength3);
                    $($stitch3top).css("stroke-dashoffset", lineLength3);
                    tween3.add(TweenMax.to($stitch3top, lineLength3 / distancePerSecond, {strokeDashoffset: 0, ease:Linear.easeNone}));
                    tween3.delay(8.7);
                var tween4 = new TimelineMax();
                    var lineLength4 = $($stitch4top)[0].getTotalLength();
                    $($stitch4top).css("stroke-dasharray", lineLength4);
                    $($stitch4top).css("stroke-dashoffset", lineLength4);
                    tween4.add(TweenMax.to($stitch4top, lineLength4 / distancePerSecond, {strokeDashoffset: 0, ease:Linear.easeNone}));
                    tween4.delay(13);
                var tween5 = new TimelineMax();
                    var lineLength5 = $($stitch5top)[0].getTotalLength();
                    $($stitch5top).css("stroke-dasharray", lineLength5);
                    $($stitch5top).css("stroke-dashoffset", lineLength5);
                    tween5.add(TweenMax.to($stitch5top, lineLength5 / distancePerSecond, {strokeDashoffset: 0, ease:Linear.easeNone}));
                    tween5.delay(11.5);

                // build scene
                var scene = new ScrollScene({triggerElement: "#trigger1", duration: 4471, tweenChanges: true}).setTween(tween).addTo(controller);
                var scene2 = new ScrollScene({triggerElement: "#trigger1", duration: 1908, tweenChanges: true}).setTween(tween2).addTo(controller);
                var scene3 = new ScrollScene({triggerElement: "#trigger1", duration: 2990, tweenChanges: true}).setTween(tween3).addTo(controller);
                var scene4 = new ScrollScene({triggerElement: "#trigger1", duration: 5470, tweenChanges: true}).setTween(tween4).addTo(controller);
                var scene5 = new ScrollScene({triggerElement: "#trigger1", duration: 7260, tweenChanges: true}).setTween(tween5).addTo(controller);
      });
```

**Comment by bodysoulkiller** [comment]
I'll add the this behavior seems only to be affecting Firefox for me.
@ajrobin: As a workaround I'll try to work with a new instance of jQuery (with the noconflict et al.)

**Comment by janpaepke** [comment]
Note that soon-to-be-released ScrollMagic 2.0 will not have a jQuery dependency anymore.

**Comment by bodysoulkiller** [comment]
Awesome ;)

@ajrobin Please note that I succeed to work with another instance of jQuery. I did however had to change the root.jQuery variable in ScrollMagic for my alias from jQuery.noConflict()
